src/main/scala/outwatch/dom/Handlers.scala

```diff
@@ -1,28 +1,44 @@
 package outwatch.dom
 
-import org.scalajs.dom.MouseEvent
-import org.scalajs.dom.raw.{ClipboardEvent, DragEvent, KeyboardEvent}
-import outwatch.Sink
-import outwatch.dom.helpers.InputEvent
-import rxscalajs.Observable
 import cats.effect.IO
+import org.scalajs.dom.{ClipboardEvent, DragEvent, KeyboardEvent, MouseEvent}
+import outwatch.dom.helpers.InputEvent
 
 /**
   * Trait containing event handlers, so they can be mixed in to other objects if needed.
   */
 trait Handlers {
-  type Handler[A] = Observable[A] with Sink[A]
-
-  def createInputHandler() = createHandler[InputEvent]()
-  def createMouseHandler() = createHandler[MouseEvent]()
-  def createKeyboardHandler() = createHandler[KeyboardEvent]()
-  def createDragHandler() = createHandler[DragEvent]()
-  def createClipboardHandler() = createHandler[ClipboardEvent]()
-  def createStringHandler(defaultValues: String*) = createHandler[String](defaultValues: _*)
-  def createBoolHandler(defaultValues: Boolean*) = createHandler[Boolean](defaultValues: _*)
-  def createNumberHandler(defaultValues: Double*) = createHandler[Double](defaultValues: _*)
-
-  def createHandler[T](defaultValues: T*): IO[Handler[T]] = Sink.createHandler[T](defaultValues: _*)
+
+  @deprecated("Use Handler.inputEvents instead", "0.11.0")
+  def createInputHandler() = Handler.create[InputEvent]
+  @deprecated("Use Handler.mouseEvents instead", "0.11.0")
+  def createMouseHandler() = Handler.create[MouseEvent]
+  @deprecated("Use Handler.keyboardEvents instead", "0.11.0")
+  def createKeyboardHandler() = Handler.create[KeyboardEvent]
+  @deprecated("Use Handler.dragEvents instead", "0.11.0")
+  def createDragHandler() = Handler.create[DragEvent]
+  @deprecated("Use Handler.clipboardEvents instead", "0.11.0")
+  def createClipboardHandler() = Handler.create[ClipboardEvent]
+
+  @deprecated("Use Handler.create[String] instead", "0.11.0")
+  def createStringHandler(defaultValues: String*) = Handler.create[String](defaultValues: _*)
+  @deprecated("Use Handler.create[Boolean] instead", "0.11.0")
+  def createBoolHandler(defaultValues: Boolean*) = Handler.create[Boolean](defaultValues: _*)
+  @deprecated("Use Handler.create[Double] instead", "0.11.0")
+  def createNumberHandler(defaultValues: Double*) = Handler.create[Double](defaultValues: _*)
+
+  @deprecated("Use Handler.create[T] instead", "0.11.0")
+  def createHandler[T](defaultValues: T*): IO[Handler[T, T]] = Handler.create[T](defaultValues: _*)
+
+
+  implicit class HandlerCreateHelpers(handler: Handler.type) {
```

Comment:
Why is this not directly in the `Handler` object in `Handler.scala`?

Reply:
I preferred to keep Handler completely generic. Also, Handler is in the outwatch package (not in outwatch.dom package), so it shouldn't "know" about dom types.